src/utils/LibBit.huff

```diff
@@ -143,6 +143,25 @@
     // Return stack:  [c]
 }
 
+/// @dev Returns 1, if the input is a power of 2.
+#define macro IS_POW_OF_2() = takes (1) returns (1) {
+    // Input stack:   [x]
+
+    dup1          // [x,x]
+    0x01          // [1,x,x]  
+    swap1         // [x,1,x] 
```

Comment:
I think that you could save one `swap`
```suggestion
     0x01          // [1,x]  
     dup2          // [x,1,x] 
```

Reply:
Nice spot! Feel free to open up a quick PR with the improvement 😄